app/scripts/modules/titus/src/titus.module.ts

```diff
@@ -61,10 +61,12 @@ module(TITUS_MODULE, [
       reader: 'titusSecurityGroupReader',
       useProvider: 'aws',
     },
-    loadBalancer: {
-      LoadBalancersTag: AmazonLoadBalancersTag,
-      useProvider: 'aws',
-    },
+    loadBalancer: defaultsDeep(
+      {
+        CreateLoadBalancerModal: TitusLoadBalancerChoiceModal,
+      },
+      CloudProviderRegistry.getProvider('aws').loadBalancer,
```

Comment:
@erikmunson I like this better, but I have mixed feelings about getting the the aws loadBalancer object from the `CloudProviderRegistry` vs importing it directly

Reply:
ugghhhh, this is not great either because now for applications with both `aws` and `titus` configured, users are now asked to pick a provider because `titus` is not 100% delegated to `aws`